Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -78,7 +78,7 @@
 QUERY_SEARCH = 'query_search'
 INCIDENT_CREATED = 'incident_created'
 
-DRILLDOWN_REGEX = r'([^\s\$]+)=(\$[^\$]+\$)|(\$[^\$]+\$)'
+DRILLDOWN_REGEX = r'([^\s\$]+)\s*=\s*["]*(\$[^\s\$]+\$)["]*|"*(\$[^\s\$]+\$)"*'
```

Comment:
as discussed
the `"` should be with the `?` 
and add the `\\`
something like that 
`([^\s\$]+)\s*=\s*"?(\$[^\s\$\\]+\$)"?|"?(\$[^\s\$\\]+\$)"?`